curriculum/challenges/english/24-b1-english-for-developers/learn-common-phrasal-verbs-and-idioms/685334cbf236decbcbb28c7c.md

```diff
@@ -0,0 +1,125 @@
+---
+id: 685334cbf236decbcbb28c7c
+title: Task 31
+challengeType: 19
+dashedName: task-31
+---
+
+<!-- (Audio) Anna: How about we put together a draft of these changes and run it by the team leads before we roll it out company-wide? -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+What does Anna suggest they do?
+
+## --answers--
+
+Prepare a version of the changes and show it to the team leads first.
+
+---
+
+Schedule a company-wide meeting to vote on the changes.
+
+### --feedback--
+
+Anna doesn't suggest checking with the whole company.
+
+---
+
+Make all the changes without informing anyone.
+
+### --feedback--
+
+Anna doesn't want to make changes without letting anyone know about them.
+
+---
+
+Send the final version directly to upper management.
+
+### --feedback--
+
+Anna doesn't talk about a final version.
+
+## --video-solution--
+
+1
+
+# --explanation--
```

Comment:
This task introduces four word or phrases. Except for `draft`, the rest are new? I usually distribute them across different tasks and add fill-in-the-blank exercises.

Reply:
Let us test how this goes with a type-19. I'll make the change in an upcoming PR.